torch/_inductor/lowering.py

```diff
@@ -2330,9 +2330,7 @@ def is_aligned(x):
 make_fallback(aten.fractional_max_pool3d_backward)
 make_fallback(aten.replication_pad1d_backward)
 make_fallback(aten.replication_pad2d_backward)
-make_fallback(aten.upsample_linear1d_backward)
 make_fallback(aten.upsample_bicubic2d_backward, require_contiguous)
```

Comment:
Why not this one?

Reply:
I'll keep linear ones for this PR and do bicubic in a follow up PR